src/package/nuspec/Microsoft.TestPlatform.TestHost.NetCore.props

```diff
@@ -12,7 +12,7 @@
       <Visible>False</Visible>
     </Content>
   </ItemGroup>
-  <ItemGroup Condition=" '$(Platform)' != 'x86' AND '$(OS)' == 'Windows_NT'" >
+  <ItemGroup Condition=" ('$(Platform)' == 'x64' OR '$(PlatformTarget)' == 'x64') AND '$(OS)' == 'Windows_NT'" >
```

Comment:
>OS [](start = 89, length = 2)

what happens when we give mulitple PlatformTargets

Reply:
With multiple PlatformTargets, dotnet build generates AnyCPU dll (then picks testhost from nuget path depending upon default architecture)